### PR TITLE
Fix EnvironmentVariableCredentialsProvider, support RamRoleArnCredentialProvider policy and RoleSessionExpiration.

### PR DESCRIPTION
**Pull request by TsinghuaDream**

### CODE REVIEW

File: aliyun-net-credentials/Provider/RamRoleArnCredentialProvider.cs [hunk 1]
```diff
@@ -46,6 +46,11 @@ public class RamRoleArnCredentialProvider : IAlibabaCloudCredentialsProvider
         {
             this.connectTimeout = config.ConnectTimeout;
             this.readTimeout = config.Timeout;
+            this.policy = config.Policy;
+            if(config.RoleSessionExpiration != 0)
```
Comment (yndu13) [inline aliyun-net-credentials/Provider/RamRoleArnCredentialProvider.cs:50]: config.RoleSessionExpiration != null的情况也判断下，这里不判断 != 0，而是应该判断 > 0

File: aliyun-net-credentials/Utils/AuthUtils.cs [hunk 2]
```diff
@@ -41,7 +41,18 @@ public static void SetPrivateKey(string key)
 
         public static string ClientType
         {
-            get { return authUtils.clientType; }
+            get
+            {
+                if (null == authUtils.clientType)
+                {
+                    authUtils.clientType = "default";
+                    return authUtils.clientType;
```
Comment (yndu13) [inline aliyun-net-credentials/Utils/AuthUtils.cs:49]: 这里不用写authUtils.clientType = "default";return authUtils.clientType;而是直接return "default"即可，不需要给authUtils.clientType赋值

**Review by yndu13 — APPROVED** [review]

File: aliyun-net-credentials/Provider/RamRoleArnCredentialProvider.cs (hunk 1 above)
Comment (TsinghuaDream) [reply aliyun-net-credentials/Provider/RamRoleArnCredentialProvider.cs]: fixed.

File: aliyun-net-credentials/Utils/AuthUtils.cs (hunk 2 above)
Comment (TsinghuaDream) [reply aliyun-net-credentials/Utils/AuthUtils.cs]: fixed.

**Pull request merged**